### DISCUSSION TITLE
Compiling pre9 with fdpp

### DISCUSSION THREAD

**Discussion by haenschen** [dosemu2/dosemu2#1758] [Q&A]
**Nevermind, ofc I need to install fdpp-dev, delete this pls :)**

~~Hi,~~

~~I'm trying to compile [2.0pre9](https://github.com/dosemu2/dosemu2/releases/tag/2.0pre9). I installed some dependencies~~

```
sudo add-apt-repository ppa:dosemu2/ppa
sudo apt update
sudo apt-get install fdpp build-essential glibc-source bison flex libsdl2-dev udev libjson-c-dev libxext-dev libslang2 gpm libasound2 fluidsynth libao-common autoconf gawk libbsd-dev
```

~~then did~~

 ```
./autogen.sh
./default-configure
```

~~which tells me that~~

```
checking for FDPP... no
configure: error: fdpp installation not found or too old
Configuration for fdpp failed, disabling
configure: Compiling without debug info...
checking for __attribute__((force_align_arg_pointer))... supported
configure: Compiling with x86 emulator
configure: Compiling with default target CPU...
configure: Compiling as PIE for x86_64...
configure: optimization set to -O2
configure: error: fdpp not available!

```

~~although installed is fdpp 1.6-1227-a067e41+202207212201~ubuntu22.04.1.~~

~~Can't I use that package with pre9?~~

**Comment by stsp** [comment]
I think you can delete yourself,
there should be "Delete discussion"
on the right side.
Anyway, what makes you to build
pre9? Maybe it will work with latest
fdpp, maybe not... Why not to use
latest devel?

**Comment by haenschen** [comment]
Unfortunately I can only "transfer discussion" and "create issue from discussion".

For years I was using Ubuntu 16.04 + [dosemu2-2.0pre6](https://github.com/dosemu2/dosemu2/releases/tag/dosemu2-2.0pre6) which is/was a very stable solution on 100+ machines. Since 16.04 is EoL I am looking for a new "stable" for my purposes.
Anyway, I was able to compile and also installed comcom32. Furthermore I found https://github.com/dosemu2/install-freedos and after installing `python3-tqdm` and doing `insfdusr` it seems like I am getting close to my former baseline but of course still need a lot of testing.

**Reply by stsp** [reply]
You are cool!
Most ppl come here with "I did this and
that and nothing works, what pile of crap
have you written?".
Glad to hear that for you things just work.

Note that current devel is actually quite
stable as I am fixing regressions for a few
months now and postponing all radical changes
(like threading and new timer stack).
